src/uct/ib/rc/verbs/rc_verbs_iface.c

```diff
@@ -436,6 +438,21 @@ unsigned uct_rc_verbs_iface_post_recv_always(uct_rc_verbs_iface_t *iface, unsign
     return count;
 }
 
+static void uct_rc_verbs_iface_vfs_refresh(void *obj)
+{
+    uct_rc_iface_t *rc_iface = (uct_rc_iface_t*)obj;
+    uct_rc_verbs_ep_t *ep;
+
+    /* Add objects for EPs */
+    ucs_list_for_each(ep, &rc_iface->ep_list, super.list) {
+        ucs_vfs_obj_add_dir(rc_iface, ep, "ep/%p", ep);
+        ucs_vfs_obj_add_ro_file(ep, ucs_vfs_show_primitive,
+                                &ep->qp->qp_num, UCS_VFS_TYPE_U32_HEX,
+                                "qp_num");
+        uct_rc_txqp_vfs_populate(&ep->super.txqp, ep);
```

Comment:
can we extract some common code with mlx5?

Reply:
done